README.md

```diff
@@ -542,23 +593,23 @@ Note that in the example above, using `utils.isValidFragment` might be unnecessa
 
 ## Development
 
-To run tests
+To run tests, use the following command:
 
 ```
 npm run test
 ```
 
-To generate test documents (for v3), you may use the script at `scripts/generate.v3.ts` and run the command
+To generate test documents (for V3), use the script at `scripts/generate.v3.ts` and run the following command:
 
 ```
 npm run generate:v3
 ```
 
 ## License
 
-[GPL-3.0](https://www.gnu.org/licenses/gpl-3.0.html)
+OpenAttestation (Verify) is under the [GPL-3.0](https://www.gnu.org/licenses/gpl-3.0.html) license.
```

Comment:
Can we update this to the apache 2.0 license

Reply:
OK, sure.